README.md

```diff
@@ -1174,3 +1175,14 @@ An example structure:
         - getter and setter for same property should appear consecutively
     - public interface methods
     - private interface methods
+
+### Macros
+
+- Inside Objective-C source, use `typeof()`.  When using the Clang compiler, or using gcc AND `GCC_ENABLE_ASM_KEYWORD=YES`, it's equivalent to the underscored versions (`__typeof()` and `__typeof()__`).
+
+- When defining a macro that includes a parameter, parenthesize any use of the parameter.
+
+    - Right
+        - `#define FOO_TIMES_TWO( x )  ( (x) * 2 )`
+    - Wrong:
+        - `#define FOO_TIMES_TWO( x )  (  x  * 2 )`
```

Comment:
Yup, there's an extra space before 'x' too.  I wanted to have the x aligned in the wrong and right.  I can't defend the alignment, so I've removed it.  In other words, I've spaced out.